### PR TITLE
COMMERCE-10877

### PR DESCRIPTION
**Pull request by Brunno0liveira**
[COMMERCE-10877](https://issues.liferay.com/browse/COMMERCE-10877)

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase [hunk 1]
```diff
@@ -1571,6 +1571,49 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-10877. Can duplicate a product and the SEO  Friendly URL match the name of the duplicated product"
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase:1574]:
```suggestion
	@description = "This is a test for COMMERCE-10877. Can duplicate a product and the SEO Friendly URL match the name of the duplicated product"
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase [hunk 2]
```diff
@@ -1571,6 +1571,49 @@ definition {
...
+	@priority = 5
+	test CanDuplicateAProductAndVerifyThatFriendlyUrlMatchesTheName {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase:1576]:
```suggestion
	test CanDuplicateProductAndVerifyFriendlyUrlMatchesProductName {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase [hunk 3]
```diff
@@ -1571,6 +1571,49 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-10877. Can duplicate a product and the SEO  Friendly URL match the name of the duplicated product"
+	@priority = 5
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase:1575]:
```suggestion
	@priority = 3
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase [hunk 4]
```diff
@@ -1571,6 +1571,49 @@ definition {
...
+				productName = "ABS Sensor",
+				publishProduct = "true");
+		}
+
+		task ("When we go to the duplicate product") {
+			CommerceProducts.openProductsAdmin();
+
+			CommerceNavigator.searchEntry(entryName = "Copy of ABS Sensor");
+
+			CommerceNavigator.gotoEntry(entryName = "Copy of ABS Sensor");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase:1606]: Use `CommerceProducts.searchAndGoToProductEntry()`

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase [hunk 5]
```diff
@@ -1571,6 +1571,49 @@ definition {
...
+
+			CommerceProducts.duplicateProduct(
+				checkProductFields = "true",
+				compileDescription = "false",
+				productCatalog = "Minium",
+				productName = "ABS Sensor",
+				publishProduct = "true");
+		}
+
+		task ("When we go to the duplicate product") {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase:1601]:
```suggestion
		task ("When the admin navigates to the duplicate product") {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase [hunk 6]
```diff
@@ -1571,6 +1571,49 @@ definition {
...
+	@description = "This is a test for COMMERCE-10877. Can duplicate a product and the SEO  Friendly URL match the name of the duplicated product"
+	@priority = 5
+	test CanDuplicateAProductAndVerifyThatFriendlyUrlMatchesTheName {
+		property portal.acceptance = "false";
+
+		task ("Given a minium site is created") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+		}
+
+		task ("And ABS Sensor is duplicated") {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase:1583]:
```suggestion
		task ("And the admin duplicates the product ABS Sensor") {
```

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:forward

**Pull request closed without merging**